transform/transformers/pck_transformer.py

```diff
@@ -134,13 +135,21 @@ def populate_period_data(self):
                 self.data['12'] = end_date.strftime("%d/%m/%Y")
 
     def round_currency_values(self):
-        """For RSI/QCAS Surveys, round the values of the currency fields.
+        """For RSI Surveys, round the values of the currency fields.
         Rounds up if the value is .5
+
+        For QCAS Surveys, round the values of the currency fields and divide
+        by 1000 (i.e., 56100 would return 56)
         """
-        if self.survey.get('survey_id') in [self.rsi_survey_id, self.qcas_survey_id]:
+        if self.survey.get('survey_id') in [self.rsi_survey_id]:
             self.data.update({k: str(Decimal(v).quantize(Decimal('1.'), ROUND_HALF_UP))
                 for k, v in self.data.items()  # noqa
```

Comment:
Since for the code below you are calling a method, I think its worth putting this into its own method also called `round_to_nearest_unit`?